frontend/components/YourWalletPage/YourAgent.tsx

```diff
@@ -59,11 +66,16 @@ const SafeAddress = ({ serviceSafe }: { serviceSafe: Safe }) => {
   );
 };
 
-const AgentTitle = ({ serviceSafe }: { serviceSafe: AgentSafe }) => {
```

Comment:
Would prefer to keep these types as they stop incorrect wallets being passed, but is fine